core/src/models/Property.ts

```diff
@@ -373,6 +380,46 @@ export class Property extends LoggedModel<Property> {
     };
   }
 
+  // --- Cache Methods --- //
+
+  static async findAllWithCache(): Promise<Property[]> {
```

Comment:
maybe something interesting to think about here with regard to "deleted" properties.

Reply:
This should be obeying the default scope, which is `ready`